crates/containerd-shim-wasm/src/sandbox/shim.rs

```diff
@@ -970,6 +970,19 @@ where
         }
 
         let i = self.get_instance(req.get_id())?;
+
+        if let Some(bundle) = i.cfg.get_bundle() {
```

Comment:
In the case of a cri pause container a new base instance is created without a bundle. I'm not sure I really understand this

Reply:
I haven't gone back and looked at the implementation (and it's been long enough I don't have it available in main memory 😄), but we pretend to start a pause container for cri since there's not really anything we can do with it.